icu4c/source/common/locid.cpp

```diff
@@ -1030,7 +1030,8 @@ Locale::forLanguageTag(StringPiece tag, UErrorCode& status)
     // uloc_forLanguageTag() to parse it if it somehow is able to detect that
     // the string actually is BCP-47. This works well for things like strings
     // using BCP-47 extensions, but it does not at all work for things like
-    // BCP-47 grandfathered tags (eg. "en-GB-oed") which are possible to also
+    // legacy language tags (marked as “Type: grandfathered” in BCP 47, e.g., "en-GB-oed")
```

Comment:
I think it'd be nicer if you kept the line wrapping of the block of text you're editing intact, instead of introducing one single line that now is longer than all the others.

Reply:
I would like to keep the whole "legacy ... marked as grandfathered ..." on one line for easier grepping for these things.
I am splitting the example to the next line though.